js/index.js

```diff
@@ -1 +1,133 @@
 // Your code goes here
+//Mouse Over / changes anchor colors
+const navAnchors = document.querySelector('.nav');
+
+navAnchors.addEventListener('mouseover', () => {
+    navAnchors.style.backgroundColor = 'gold';
+    console.log('gold nav bar')
+    
+});
+
+navAnchors.addEventListener('mouseleave', () => {
+    navAnchors.style.backgroundColor = 'white';
+    console.log('default nav bar')
+});
+
+navAnchors.addEventListener('click', (event) => {
+    event.preventDefault();
+});
+
+
+//Key Down / Press any key for a secret message for any plagerizers
+const pageBody = document.querySelector('body');
+const noPlagerismFont = document.querySelector('.footer p');
+
+pageBody.addEventListener('keydown', () => {
+    noPlagerismFont.textContent = 'Copyright Fun Bus 2019. https://www.copyright.gov/help/faq/definitions.html';
```

Comment:
Another reason not to plagiarise...

![image](https://user-images.githubusercontent.com/8473052/59236614-0d54f300-8bc5-11e9-8f6d-5e57cf6d28c8.png)